spec/dummy/app/views/components/docs/test-component-with-block.yml

```diff
@@ -0,0 +1,7 @@
+name: Test component with block
+description: A test component that takes a block
+examples:
+  default:
+    data:
+      block: |
```

Comment:
Block is a good shout, but what do you think about naming it something like 'children'?

If you understand blocks this makes sense, but children may make more sense for people who do not in the context of interfaces.

Not sure, what do you think?

Reply:
I prefer `block` or `yields`.

Reply:
I'm not 100% sold on `children`, we're leveraging the `ActionView::Partial#render` method's ability to accept a block and yield it. I agree though, that with little ruby experience the name `block` is not intuitive.